services/ui-src/src/components/layout/Header.tsx

```diff
@@ -183,4 +175,11 @@ const sx = {
   closeIcon: {
     width: "2rem",
   },
+  leaveFormText: {
```

Comment:
Idea: what do you think of moving all of these styles into a new variant for the link in the `theme.ts` file? So instead of using the `unstyled` variant, we could add a new one like `unstyledButton` (or whatever sounds good).

```js
Link: {
  variants: {
  // all the other variants would be above... 
    unstyledButton: {
          color: "palette.primary",
          border: "1px solid",
          padding: ".5rem 1rem",
          borderRadius: "5px",
          fontWeight: "bold",
          textDecoration: "none",
          _visited: {
            color: "palette.primary",
          },
          ":hover, :visited:hover": {
            color: "palette.primary_darker",
            textDecoration: "none",
          },
        },

  }
}
```

Alternatively, we could move this block up to the `leaveFormLink` property but might have to do some additional tinkering to override some design system component styling